webui/Makefile

```diff
@@ -24,7 +24,6 @@ check-%:
 .PHONY: get-deps
 get-deps:
 	$(MAKE) -C react $@
-	$(MAKE) -C tests $@
```

Comment:
Uh, just checking this is intentional?

Reply:
yes. it comes with downloading chromium, about 140MB, and we just don't need it on normal local installs.

Reply:
it will have some effect on the make check targets and I just realized we're missing a ci step to lint the tests. I'll put this line back and address it with the ci job in a separate pr.